### PR TITLE
minor changes to project

### PR DESCRIPTION
**Pull request by grillazz**
- python libraries update: Django 2.0.2 and django-pyodbc-azure 2.0.1.0
- align project files to Django 2.0 layout 
- Microsoft sample AdventureWorksDW2017 added as curl download to Dokcerfile
- update project settings.py with AdventureWorksDW2017 as default database
- update docker-compose.yml with AdventureWorksDW2017 as default database
- AdventureWorksDW2017 models via inspecdb command vith small fixes

### CODE REVIEW

File: samples/development-frameworks/SqlServerOnDocker/docker-compose.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ services:
       SA_PASSWORD: "Alaska2017"
       ACCEPT_EULA: "Y"
     healthcheck:
-      test: sqlcmd -S db1.internal.prod.example.com -U SA -P 'Alaska2017' -Q 'select 1'
+      test: sqlcmd -S db1.internal.prod.example.com -U SA -P 'Alaska2017' -Q 'select 1 from AdventureWorksDW2017'
```
Comment (uc-msft) [inline samples/development-frameworks/SqlServerOnDocker/docker-compose.yml]: Seems incorrect. Is there a table called AdventureWorksDW2017?

**Comment by grillazz** [comment]
@uc-msft Thanks. That is fixed now.

**Comment by grillazz** [comment]
@LuisBosquez , @uc-msft is there any chance to close it ? I would like to add similar project example but on AGs.

**Comment by grillazz** [comment]
@LuisBosquez , @uc-msft should I remove this PR ?

**Comment by grillazz** [comment]
@uc-msft any chance for approval ?

File: samples/development-frameworks/SqlServerOnDocker/docker-compose.yml (hunk 1 above)
Comment (grillazz) [reply samples/development-frameworks/SqlServerOnDocker/docker-compose.yml]: @uc-msft  any feedback ?

Comment (grillazz) [reply samples/development-frameworks/SqlServerOnDocker/docker-compose.yml]: conflicts resolved as well

**Pull request merged**